docs/README-develop-best-practices.md

```diff
@@ -206,6 +207,42 @@ The following commands should be preferred over `show [feature]` commands since
 
 YAML allows various synonyms for `true` and `false` such as `yes` and `no`, but for consistency and readability (especially to users more familiar with Ruby than with YAML), we recommend using `true` and `false` rather than any of their synonyms.
 
+### <a name="yaml11">Y11: Use YAML anchors and aliases to avoid redundant entries.
+
+Use the standard YAML functionality of [node anchors](http://www.yaml.org/spec/1.2/spec.html#id2785586) and [node aliases](http://www.yaml.org/spec/1.2/spec.html#id2786196) to avoid redundant entries. In other words, instead of:
+
+```yaml
+  vn_segment_vlan_based:
+   # MT-lite only
+   N3k:
+     kind: boolean
+     config_get: 'show running section feature'
+     config_get_token: '/^feature vn-segment-vlan-based$/'
+     config_set: 'feature vn-segment-vlan-based'
+     default_value: false
+   N9k:
+     # same as N3k
+     kind: boolean
+     config_get: 'show running section feature'
+     config_get_token: '/^feature vn-segment-vlan-based$/'
+     config_set: 'feature vn-segment-vlan-based'
+     default_value: false
+```
+
+instead you can do:
+
+```yaml
+  vn_segment_vlan_based:
+   # MT-lite only
+   N3k: &vn_segment_vlan_based_N3k
+     kind: boolean
+     config_get: 'show running section feature'
+     config_get_token: '/^feature vn-segment-vlan-based$/'
+     config_set: 'feature vn-segment-vlan-based'
+     default_value: false
+   N9k: *vn_segment_vlan_based_N3k
```

Comment:
I like the readme writeups. 

Just one last suggestion on the anchor ref names themselves: it might be better (at least in the MT cases) to use an abstract tag instead of referencing the plat type, e.g:

```
N3k: &mt_lite_vn_segment_vlan_based
 ...
N9k: *mt_lite_vn_segment_vlan_based

```

```
N3k: &mt_lite_mapped_vlan
 ...
N9k: *mt_lite_mapped_vlan
```

```
N3k: &mt_lite_destroy
 ...
N9k: *mt_lite_destroy
```